http_status/lib/rfc7231.status_codes

```diff
@@ -33,6 +33,7 @@ Code Reason-Phrase
 415 Unsupported Media Type
 416 Range Not Satisfiable
 417 Expectation Failed
+422 Unprocessable Entity
```

Comment:
Not part of https://tools.ietf.org/html/rfc7231#section-6.1

Should be a new file I think. Is there a rfc that defines 422